source/guides/community-for-mattermost.rst

```diff
@@ -44,38 +46,32 @@ Getting started
 
 - **Stay informed**: Follow the `~release-announcements <https://community.mattermost.com/core/channels/release-announcements>`_ channel to keep up with the latest news about upcoming releases and updates from the Mattermost team.
 
 - **Contribute**: If you're interested in contributing to Mattermost, join the `~developers <https://community.mattermost.com/core/channels/developers>`_ channel to connect with other contributors.
 
-- **Share feedback**: Your ideas and feedback are valuable! Share your thoughts in the `~user-feedback <https://community.mattermost.com/core/channels/user-feedback>`_ channel.
+- **Share feedback**: Your Mattermost ideas and feedback are valuable! Share your thoughts in the `~user-feedback <https://community.mattermost.com/core/channels/user-feedback>`_ channel.
 
-Roadmap: Upcoming Features for Microsoft Teams Integration
-----------------------------------------------------------
-
-We are continually enhancing the integration experience between Mattermost and Microsoft Teams. Here's a glimpse of what's coming next:
-
-1. **Entra-Based SSO**:
-
-   - Seamless and secure single sign-on for organizations using Microsoft Entra ID.
-   - Benefits: Simplifies user authentication and enhances security with enterprise-grade identity management.
-
-2. **Activity Feed Notifications for Mentions in Mattermost**:
-
-   - Stay updated with real-time notifications in your Microsoft Teams activity feed whenever someone mentions you in Mattermost.
-   - Benefits: Ensures you never miss critical updates, even while collaborating within Microsoft Teams.
+Key features
+------------
 
-3. **Support for Multiple Mattermost Instances in Teams Tabs**: 
+- **Direct Access**: Access the Community for Mattermost directly from a tab without switching applications or opening a browser.
+- **Seamless Integration**: Experience the full functionality of Community for Mattermost within a familiar interface.
+- **Real-time Collaboration**: Engage with the Community for Mattermost in real-time discussions on product features, technical questions, and best practices.
+- **Product Support**: Get help from both Mattermost staff and experienced community members.
+- **Contribute to Development**: Participate in discussions that shape the future direction of Mattermost products.
+- **Knowledge Sharing**: Learn implementation strategies and best practices from a diverse community of users.
+- **Stay Updated**: Keep up with the latest Mattermost announcements, updates, and roadmap information.
 
-   - View and access multiple Mattermost instances directly from different tabs in Microsoft Teams.
-   - Benefits: Streamlined workflows for organizations with multiple Mattermost deployments or diverse team setups.
+Benefits
+--------
 
-4. **Unread Badge and Unread Counter Badge**: 
+- Seamlessly communicate with the Mattermost Community from Microsoft Teams and Outlook using application tabs.
```

Comment:
```suggestion
- Seamlessly communicate with the Mattermost community using application tabs.
```

Reply:
This line seems redundant.  We already said this in the Key Features section.  Thoughts?